pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -325,6 +325,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats `json:"antreaClusterNetworkPolicies,omitempty" protobuf:"bytes,3,rep,name=antreaClusterNetworkPolicies"`
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats `json:"antreaNetworkPolicies,omitempty" protobuf:"bytes,4,rep,name=antreaNetworkPolicies"`
+	// List of PodMulticast statistics collected from the Node.
+	MulticastStats []MulticastStats `json:"multicastStats,omitempty" protobuf:"bytes,5,rep,name=multicastStats"`
+}
+
+// MulticastStats contains the multicast groups that a Pod has joined.
+type MulticastStats struct {
```

Comment:
Do we plan to include more per pod data to this struct? I wonder why it is pod level but not group level.

Reply:
Updated. 

We do have plan to add more per pod data to this struct because there will be a requirement "showing pod registered statically an interest in a specific multicast group"

After second thought, I think the group level struct is still better for aggregation in the controller. We may use an extra struct, maybe `StaticMulticastStats` for the future requirements I mentioned above.